setup.py

```diff
@@ -36,11 +36,17 @@ def readme():
         'lxml'
     ],
     extras_require={
+        'docs': [
+            'sphinx',
+            'sphinx-argparse',
+            'sphinx_rtd_theme'
+        ],
         'development': [
             'gitpython',
             'P4',
             'sphinx',
-            'sphinx-argparse'
+            'sphinx-argparse',
+            'sphinx_rtd_theme'
```

Comment:
Does it generate docs with rtd theme when run locally?
Can we reference one extra from another?

Reply:
Yes, seems so.